.web.yml

```diff
@@ -1,5 +1,5 @@
 tutorials:
-  - title: Snorkel Intro Tutorial
+  - title: Introduction to Labeling Functions
```

Comment:
do we want "Introduction to" or "Intro to"? And we should be consistent. I vote "Intro to", fits in the website boxes better

Reply:
yes you're right